core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.css

```
@@ -0,0 +1,80 @@
+  overflow: hidden;
+  padding: 10px 16px 10px 16px;
+  text-align: left;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+}
+.oppia-thumbnail-uploader-container {
+  min-height: 300px;
+}
+.oppia-form-error {
+  margin-top: 15px;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)